demos/prefsFramework/html/SimplifyPanelTemplate.html

```diff
@@ -1,11 +1,10 @@
 <h2><span class="fl-icon-simplify"></span><span class="demo-prefsEditor-simplify-label heading-text"></span></h2>
-
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="simplify-choice" class="demo-prefsEditor-simplify" />
     <label for="simplify-choice">
+    	<span class="demo-prefsEditor-simplify-choice-label block"></span>
```

Comment:
the styling classes should be in a fl- namespace if they are part of the component or a demo- namespace if they are part of the demo.